fcsa_service_request-1.0.0-resolved.json

```diff
@@ -0,0 +1,668 @@
+{
+  "openapi": "3.0.0",
+  "info": {
+    "title": "FCSA Service Request",
+    "description": "FCSA Service Request",
+    "contact": {
+      "email": "dee@opencitieslab.org"
+    },
+    "license": {
+      "name": "Apache 2.0",
+      "url": "http://www.apache.org/licenses/LICENSE-2.0.html"
+    },
+    "version": "2.1"
+  },
+  "servers": [
+    {
+      "url": "https://qaeservices1.capetown.gov.za/coct/api/zcur-guest/login",
+      "description": "FCSA Service Request API"
+    }
+  ],
+  "tags": [
```

Comment:
I'm not familiar with the tags semantic in OpenAPI - what do they do generally, and how are you using them here?

Reply:
I have updated the ReadMe that provides a definition of tags, components, etc.